packages/lib-sourcify/src/Verification/VerificationTypes.ts

```diff
@@ -40,3 +56,67 @@ export type VerificationStatus =
   | 'extra-file-input-bug'
   | 'error'
   | null;
+
+export interface VerificationExport {
+  address: string;
+  chainId: number;
+  status: {
+    runtimeMatch: VerificationStatus;
+    creationMatch: VerificationStatus;
+  };
+  onchainRuntimeBytecode?: string;
+  onchainCreationBytecode?: string;
+  transformations: {
+    runtime: {
+      list: Transformation[];
+      values: TransformationValues;
+    };
+    creation: {
+      list: Transformation[];
+      values: TransformationValues;
+    };
+  };
+  deploymentInfo: {
+    blockNumber?: number;
+    txIndex?: number;
+    deployer?: string;
+    txHash?: string;
+  };
+  libraryMap: {
+    runtime?: StringMap;
+    creation?: StringMap;
+  };
+  compilation: {
+    language: CompilationLanguage;
+    compilationTarget: CompilationTarget;
+    compilerOutput: {
+      // The export should not include the AST object to reduce the size
+      sources?: Record<string, { id: number }>;
+    };
+    contractCompilerOutput: {
+      abi?: Abi;
+      userdoc?: Userdoc;
+      devdoc?: Devdoc;
+      storageLayout?: StorageLayout;
+      evm: {
+        bytecode: {
+          sourceMap?: string;
+          linkReferences?: LinkReferences;
+        };
+        deployedBytecode: {
+          sourceMap?: string;
+          linkReferences?: LinkReferences;
+          immutableReferences?: ImmutableReferences;
+        };
+      };
+    };
```

Comment:
I'm wondering here can we use `SolidityOutputContract | VyperOutputContract` as type?

Reply:
They contain more output than we use. Therefore, I minimized it that way.

Reply:
Ok for now let's keep it like this, if in the future someone needs more fields we can add them